packages/terra-clinical-site/package.json

```diff
@@ -69,9 +66,17 @@
     "node-sass": "^4.5.2",
     "postcss-loader": "^1.3.3",
     "raw-loader": "^0.5.1",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "redux": "^3.6.0",
+    "react-redux": "^5.0.4",
+    "react-router": "^3.0.5",
     "sass-loader": "^6.0.3",
     "style-loader": "^0.16.1",
-    "terra-i18n-plugin": "^0.x",
```

Comment:
https://github.com/cerner/terra-clinical/pull/93/commits/1f458082499a49f79f0f5757ab4c0f42d8d3558e
Moved to devDependency